tests/run-make/core-ffi-typecheck-clang/rmake.rs

```diff
@@ -0,0 +1,194 @@
+//@ needs-force-clang-based-tests
+// This test checks that the clang defines for each target allign with the core ffi types defined in
+// mod.rs. Therefore each rust target is queried and the clang defines for each target are read and
```

Comment:
Suggestion: (after you finish the `core/ffi` changes) can you specify an *exact* path to the file in question?